test/SILOptimizer/moveonly_closure_lifetime_fixup_address_only_borrowed_captures.swift

```diff
@@ -0,0 +1,64 @@
+// RUN: %target-swift-frontend -emit-sil %s | %FileCheck %s
+
+// When ClosureLifetimeFixup promotes a closure to the stack, we should
```

Comment:
Can you add a SIL test case that shows this behavior? I would be fine with a single FileCheck test case that just shows the behavior in action.